src/main/java/de/hpi/bpt/argos/util/RestEndpointUtilImpl.java

```diff
@@ -138,7 +138,7 @@ public String executeRequest(Logger logger, Request request, Response response,
 	 */
 	@Override
 	public void logReceivedRequest(Logger logger, Request request) {
-		logger.info(String.format("received request: '%1$s' -> %2$d bytes of content", request.uri(), request.contentLength()));
+		logger.info(String.format("%1$s\t->\t'%2$s' -> %3$d bytes of content", request.ip().toString(), request.uri(), request.contentLength()));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') "ip" returns a string, there's no need to call "toString()". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1858)